fw-child/resources/js/map.js

```diff
@@ -1350,39 +1351,52 @@
         options = plugin.options,
         item = plugin.item;
 
-      // todo: implement behaviour for building_climate_zones
-      let layer_name = $(document).cdc_app(
-        'maps.get_layer_name',
-        options.query,
+      let default_scheme_element = item.find(
+        '#display-scheme-select .dropdown-item[data-scheme-id="default"]',
       );
-      $.getJSON(
-        geoserver_url +
+
+      if (special_variables.hasOwnProperty(options.var_data.slug)) {
+        default_scheme_element.data('scheme-colours', special_variables[options.var_data.slug].colormap.colours);
+        default_scheme_element.data('scheme-quantities', special_variables[options.var_data.slug].colormap.quantities);
```

Comment:
Create a `special_var` variable, like in line 1409, just to lighten the first 2 lines of the if:
```suggestion
      if (special_variables.hasOwnProperty(options.var_data.slug)) {
        const special_var = special_variables[options.var_data.slug];
        default_scheme_element.data('scheme-colours', special_var.colormap.colours);
        default_scheme_element.data('scheme-quantities', special_var.colormap.quantities);
```